### ISSUE TITLE
all of the S3 function do not work

### ISSUE THREAD

**Issue by jingpengw** [JuliaCloud/AWS.jl#82]
suddenly, all of the s3 function get an S3Error. Is that due to S3 outage again?
Could anyone else run the `tests3.jl`?

```
S3Response
content_length: 0
date: Tue, 14 Mar 2017 20:32:58 GMT
server: AmazonS3
eTag: 
http_code: 403
delete_marker: false
id_2: 
request_id: 
version_id: 
headers: Pair{AbstractString,AbstractString} Dict
    Transfer-Encoding => chunked
    status_code => 403
    http_major => 1
    x-amz-request-id => **********
    Content-Type => application/xml
    Date => Tue, 14 Mar 2017 20:32:58 GMT
    http_minor => 1
    Keep-Alive => 1
    Server => AmazonS3
    x-amz-id-2 => ****************=
obj: AWS.S3.S3Error("SignatureDoesNotMatch","The request signature we calculated does not match the signature you provided. Check your key and signing method.","********=","********")
```

**Comment by jingpengwu** [comment]
I think I introduced a bug due to my PR. 
https://github.com/JuliaCloud/AWS.jl/pull/73

I just tested my old code, the `tests3.jl` works. Something is not compatible between these two branches.

**Comment by mdpradeep** [comment]
I had tested `tests3.jl` (along with `testec2.jl` and `testsqs.jl`)  before I merged https://github.com/JuliaCloud/AWS.jl/pull/73 and it worked. I had to add the `LocationConstraint` stuff to get `tests3.jl` to work (https://github.com/JuliaCloud/AWS.jl/pull/81)

**Comment by jingpengwu** [comment]
Thanks for the response. I got S3Error for all the operations including `list_bucket`!

**Comment by jingpengwu** [comment]
I used my PR branch frequently, works fine. so it might be introduced by the other PR (https://github.com/JuliaCloud/AWS.jl/pull/79)

**Comment by tanmaykm** [comment]
Should be addressed with #90 now.
Please reopen if this is still an issue for you.
